org.eclipse.xtext.util/src/org/eclipse/xtext/util/MergeableManifest.java

```diff
@@ -50,38 +50,47 @@
 	public static final Attributes.Name BUNDLE_LOCALIZATION = new Attributes.Name("Bundle-Localization");
 	public static final Attributes.Name BUNDLE_ACTIVATOR = new Attributes.Name("Bundle-Activator");
 
-	private static final String LINEBREAK = "\r\n";
+	private static final String LINEBREAK = Strings.newLine();
```

Comment:
this filed is only used by deprecated methods, right? Then it should be deprecated itself.